### PR TITLE
Limit tests

### PR DESCRIPTION
**Pull request by ultimateboy**
Fixes deis/workflow-e2e#196
Fixes deis/workflow-e2e#198
Fixes deis/workflow-e2e#199
Fixes deis/workflow-e2e#200
Fixes deis/workflow-e2e#201

### CODE REVIEW

File: cmd/users_test.go [hunk 1]
```diff
@@ -67,3 +67,46 @@ test
...
+					"is_active": true,
+					"date_joined": "2014-10-19T22:01:00.601Z",
+					"groups": [],
+					"user_permissions": []
+				}
+			]
+		}`)
+	})
+
+	err = cmdr.UsersList(-1)
```
Comment (Joshua-Anderson) [inline cmd/users_test.go]: Do you want `1` here instead of `-1`?

**Comment by Joshua-Anderson** [comment]
One minor thing I noticed, otherwise LGTM.

**Comment by ultimateboy** [comment]
Nice catch @Joshua-Anderson. I'm going to spend a bit of time investigating why that was passing as it was written. But fixed.

**Comment by Joshua-Anderson** [comment]
I'm thinking it was passing because the SDK returned `1` item but the `count` on the object was set to two. The code doesn't look at the requested limit, it only looks at the effective limits (returned items vs total count).

**Comment by Joshua-Anderson** [comment]
Could you possibly add limits tests for `KeysList()`?

**Comment by Joshua-Anderson** [comment]
I also noticed that we still need tests for  limits on `app.Logs()`. If you want to want to wait and do these in a different PR that's perfectly fine as well.

**Comment by ultimateboy** [comment]
Thanks @Joshua-Anderson. Somehow missed the keys test. Added.

**Comment by Joshua-Anderson** [comment]
After my changes to codecov, you need to rebase. Sorry!

**Pull request merged**